### PR TITLE
feat: add function `get_cached_related_obj()`

### PR DESCRIPTION
**Pull request by robinzachmann**

### CODE REVIEW

File: ambient_toolbox/utils/model.py [hunk 1]
```diff
@@ -29,3 +29,41 @@ def object_to_dict(obj, blacklisted_fields: list = None, include_id: bool = Fals
...
+
+    The only difference is the usage of `select_related()`, which can easily be forgotten.
+
+    Usage:
+    get_cached_related_obj(foo, 'bar')
+
+    This will result in a KeyError instead of a silent sub-query if the field is not cached
+    via `select_related()` or `prefetch_related()`.
+    """
+    fields_cache = obj._state.fields_cache  # warning: this is an undocumented feature of django
```
Comment (GitRon) [inline ambient_toolbox/utils/model.py:58]: nitpick: i prefer inline comments to be above the LOC

**Review by GitRon — CHANGES_REQUESTED** [review]
Apart from the nitpick, very nice 👌

File: ambient_toolbox/utils/model.py [hunk 2]
```diff
@@ -29,3 +29,41 @@ def object_to_dict(obj, blacklisted_fields: list = None, include_id: bool = Fals
             valid_data[key] = value
 
     return valid_data
+
+
+def get_cached_related_obj(obj: Model, related_field_name: str, silently_return_none: bool = False):
```
Comment (GitRon) [inline ambient_toolbox/utils/model.py:34]: issue: could you add the relevant parts of the docstring to the docs, please? ❤️

File: ambient_toolbox/utils/model.py (hunk 1 above)
Comment (robinzachmann) [reply ambient_toolbox/utils/model.py]: not sure if that makes it more readable, but if makes you happy ... 🫶

File: ambient_toolbox/utils/model.py (hunk 2 above)
Comment (robinzachmann) [reply ambient_toolbox/utils/model.py:34]: done

Comment (GitRon) [reply ambient_toolbox/utils/model.py:35]: ❤️

**Review by GitRon — APPROVED** [review]
Great work, thx!

**Pull request merged**